frontend/src/app/components/main-menu/main-menu-toggle.service.ts

```diff
@@ -40,7 +41,8 @@ export class MainMenuToggleService {
   private elementWidth:number;
   private readonly localStorageKey:string = 'openProject-mainMenuWidth';
   private readonly defaultWidth:number = 230;
-  private readonly currentProject:CurrentProjectService = this.injector.get(CurrentProjectService);
```

Comment:
For whatever reason this was again called in line [79](https://github.com/opf/openproject/blob/0d5d295fec5e57b4542a818c3f681c2be34b856b/frontend/src/app/components/main-menu/main-menu-toggle.service.ts#L79). I guess the second one can be removed, too.